sensorlib/Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM ubuntu:focal
-WORKDIR /home/CarlaCDASimAdapter/
-COPY ./src/ /home/CarlaCDASimAdapter/src
-COPY requirements.txt /home/CarlaCDASimAdapter/
-COPY ./config/ /home/CarlaCDASimAdapter/config
+
 ## Install Dependencies
 RUN apt-get update
 RUN apt-get install -y gfortran wget python3 python3-pip libtiff5 libjpeg-dev libpng16-16
+WORKDIR /home/CarlaCDASimAdapter/
 RUN wget https://github.com/usdot-fhwa-stol/carma-carla-integration/raw/develop/PythonAPI/carla/dist/carla-0.9.10-py3.7-linux-x86_64.egg
+
 ENV LOAD_CARLA_EGG=True
 ENV CARLA_VERSION=0.9.10
 ENV CARLA_EGG_DIR="/home/CarlaCDASimAdapter/"
-RUN pip3 install -r requirements.txt
+
 WORKDIR /home/CarlaCDASimAdapter/
-CMD [ "python3", "/home/CarlaCDASimAdapter/src/CarlaCDASimAdapter.py", "--carla-host", "127.0.0.1", "--xmlrpc-server-host", "127.0.0.1",  "--sensor-config-file", "../config/simulated_sensor_config.yaml", "--noise-model-config-file", "../config/noise_model_config.yaml" ]
\ No newline at end of file
+COPY ./requirements.txt /home/CarlaCDASimAdapter/
+RUN pip3 install -r requirements.txt
+
+COPY ./src/ /home/CarlaCDASimAdapter/src
+COPY ./config/ /home/CarlaCDASimAdapter/config
+
+CMD [ "python3", "/home/CarlaCDASimAdapter/src/CarlaCDASimAdapter.py", "--carla-host", "172.2.0.2", "--xmlrpc-server-host", "127.0.0.1",  "--sensor-config-file", "../config/simulated_sensor_config.yaml", "--noise-model-config-file", "../config/noise_model_config.yaml" ]
```

Comment:
Do we need this command line if the command is in docker-compose.yml?

Reply:
Let's remove this CMD

Reply:
Ok. It can be in either place, if in the Dockerfile then the container will automatically launch the command otherwise it needs to be called externally. There's not too much difference for our use case.

Reply:
it is being called externally in my other image, so that if people need to give a different argument, they can. So let's remove it.